src/main/resources/anet-schema.yml

```diff
@@ -476,6 +476,15 @@ properties:
                 title: The long date/time format if `engagementsIncludeTimeAndDuration` is `true`
                 description: Used to show the report engagement date/time in the user interface; should be easy to read, e.g. "Sunday, 6 December 1998 @ 15:30"; see https://momentjs.com/docs/#/displaying/format/ for format specifiers.
 
+  calendarOptions:
+    type: object
+    additionalProperties: false
+    required: [attendeesType]
+    properties:
+      attendeesType:
+        type: string
```

Comment:
I would force this to be either `advisor` or `interlocutor`, so specifying anything else would invalidate the dictionary.